mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/mule/steps/salesforce/CreateOperation.java

```diff
@@ -0,0 +1,147 @@
+      if (children.size() > 0) {
+        Element mule4Headers = new Element("headers", SalesforceConstants.MULE4_SALESFORCE_NAMESPACE);
+        children.stream()
+          .forEach(header -> {
+            mule4Headers.addContent(
+              new Element("header", SalesforceConstants.MULE4_SALESFORCE_NAMESPACE)
+                .setAttribute("key", header.getText())
+                .setAttribute("value", header.getText()));
+            });
+        mule4CreateOperation.addContent(mule4Headers);
+      }
+    });
+
+    Optional<Element> objects =
+        Optional.ofNullable(mule3CreateOperation.getChild("objects", SalesforceConstants.MULE3_SALESFORCE_NAMESPACE));
+
+    objects.ifPresent(records -> {
+      Optional.ofNullable(records.getAttributeValue("ref")).ifPresent(value -> {
+        Element recordsChild = new Element("records", SalesforceConstants.MULE4_SALESFORCE_NAMESPACE);
+
+        String expression = expressionMigrator.migrateExpression(value, true, records);
+        recordsChild.setContent(new CDATA(expression));
+
+        mule4CreateOperation.addContent(recordsChild);
+      });
+
+      List<Element> children = records.getChildren();
+      if (children.size() > 0) {
+        String transformBody = children.stream()
+            .map(object -> object.getChildren().stream()
+                .map(innerObject -> innerObject.getAttributeValue("key") + " : \"" + innerObject.getText() + "\"")
+                .collect(Collectors.joining(",\n")))
+            .collect(Collectors.joining("\n},\n{"));
+
+
+        getApplicationModel().addNameSpace(CORE_EE_NAMESPACE, EE_NAMESPACE_SCHEMA, mule3CreateOperation.getDocument());
+        Element element = new Element("transform");
+        element.setName("transform");
+        element.setNamespace(CORE_EE_NAMESPACE);
+        element.removeContent();
+        element.addContent(new Element("message", CORE_EE_NAMESPACE)
+            .addContent(new Element("set-payload", CORE_EE_NAMESPACE)
+                .setText("%dw 2.0 output application/json\n---\n[{\n" + transformBody + "\n}]")));
+
+        XmlDslUtils.addElementBefore(element, mule3CreateOperation);
+      }
+    });
+
+    XmlDslUtils.addElementAfter(mule4CreateOperation, mule3CreateOperation);
+    mule3CreateOperation.getParentElement().removeContent(mule3CreateOperation);
+  }
+
+  private void setAttributes(Element mule3CreateOperation, Element mule4CreateOperation, MigrationReport report) {
+    String docName = mule3CreateOperation.getAttributeValue("name", SalesforceConstants.DOC_NAMESPACE);
+    if (docName != null) {
+      mule4CreateOperation.setAttribute("name", docName, SalesforceConstants.DOC_NAMESPACE);
+    }
+
+    String configRef = mule3CreateOperation.getAttributeValue("config-ref");
+    if (configRef != null && !configRef.isEmpty()) {
+      mule4CreateOperation.setAttribute("config-ref", configRef);
+    } else {
+      report.report("salesforce.create", mule4CreateOperation, mule4CreateOperation);
```

Comment:
I don't think it's necessary to add an entry to the report because this field wasn't present. 
We tend to add entries to the report whether we cannot perform a migration because there is no way to migrate it or they could use something else to achieve an action.